lib/tasks/asset_compile.rake

```diff
@@ -0,0 +1,23 @@
+require 'json'
+
+Rake::Task['assets:precompile'].enhance do
+  Rake::Task['asset_precompile:create_non_digest_assets'].invoke
+end
+
+namespace :asset_precompile do
+  task :create_non_digest_assets do
+    relative_asset_path = 'public/static'
+    manifest_path = Dir.glob(File.join(Rails.root, relative_asset_path, '.sprockets-manifest-*.json')).first
+
+    manifest_data = JSON.load(File.new(manifest_path))
+
+    manifest_data["assets"].each do |logical_path, digested_path|
+      if logical_path.end_with?('.js')
+        full_digested_path = File.join(Rails.root, relative_asset_path, digested_path)
+        full_nondigested_path = File.join(Rails.root, relative_asset_path, logical_path)
+
+        FileUtils.copy_file full_digested_path, full_nondigested_path, true
```

Comment:
Could we do a link here, instead of copying it?

Reply:
Done.